orbit/pkg/execuser/execuser_darwin.go

```diff
@@ -41,7 +41,8 @@ func run(path string, opts eopts) (lastLogs string, err error) {
 		}
 	}
 
-	cmd := exec.Command("/usr/bin/open", arg...)
+	arg = append([]string{"-u", opts.user, "/usr/bin/open"}, arg...)
+	cmd := exec.Command("sudo", arg...)
```

Comment:
I'm confused as to why we didn't use `sudo` from the get go and how this is working today... (given root is executing `/usr/bin/open`...)

Reply:
According to my research, `open` uses the Launch Services API to launch apps as the GUI user.